invokeai/frontend/web/src/features/parameters/hooks/useRecallParameters.ts

```diff
@@ -65,6 +65,7 @@ import {
   setSeed,
   setSteps,
   setWidth,
+  vaeSelected,
```

Comment:
This should function should be renamed to `setVae` or `setVaeModel` for consistency sake. I can do that if the reviewers don't mind, but to keep the scope of this PR small, I am not jumping the gun.

Reply:
It's OK to leave it, actually the other functions should be named to [model events](https://redux.js.org/style-guide/#model-actions-as-events-not-setters), but we only started doing that recently and it's not a hard rule